source/tests/unit/xnet_converters_tests.cpp

```diff
@@ -601,6 +608,25 @@ TEST(XnetConvertersTests, IPv4AddrOutputConverter_ConvertToGrpc_ConvertsToAddres
   EXPECT_EQ(ADDRESS, grpc_addr.addr());
 }
 
+TEST(XnetConvertersTests, StringSockOptData_GetSize_FetchesSizeFromLibrary)
+{
+  constexpr auto DEFAULT_SOCK_OPT_STRING_SIZE = 255;
+  auto ACTUAL_SOCK_OPT_STRING_SIZE = 12;
+  NiXnetSocketMockLibrary library;
+  auto storage = allocate_output_storage<void*, SockOptData>(&library, OptName::OPT_NAME_SO_BIND_TO_DEVICE);
+  EXPECT_EQ(DEFAULT_SOCK_OPT_STRING_SIZE, storage.string_length);
+  // Expect call GetSockOpt(socket, level, opt_name, nullptr, &string_length)
```

Comment:
not sure if this comment provides a lot, it is obvious we expect the call from line 623

Reply:
removed